tests/xharness/TestLabel.cs

```diff
@@ -0,0 +1,129 @@
+using System;
+using System.Diagnostics;
+using System.Linq;
+using Mono.Unix.Native;
+
+#nullable enable
+
+namespace Xharness {
+	[AttributeUsage (AttributeTargets.Field)]
+	public class LabelAttribute : Attribute {
+		public string Label { get; }
+
+		public LabelAttribute (string label)
+		{
+			Label = label;
+		}
+	}
+
+	[Flags]
+	public enum TestLabel {
+		//{ "all", false },
+		[Label ("all")]
```

Comment:
What is the reason for the Label attributes here?

Reply:
Oh is it so we can retrieve the label if we need this with code like this below?
```
enumType.GetField(name).GetCustomAttributes(false)
				.OfType<LabelAttribute>().SingleOrDefault();
```

Reply:
Correct, so that we can convert easily from an enum to a string that is not the Enum.ToString()